Library/Homebrew/cmd/vendor-install.sh

```diff
@@ -17,9 +17,9 @@ if [[ -n "$HOMEBREW_MACOS" ]]
 then
   if [[ "$HOMEBREW_PROCESSOR" = "Intel" ]]
   then
-    ruby_URL="$HOMEBREW_BOTTLE_DOMAIN/bottles-portable-ruby/portable-ruby-2.6.3.mavericks.bottle.tar.gz"
-    ruby_URL2="https://github.com/Homebrew/homebrew-portable-ruby/releases/download/2.6.3/portable-ruby-2.6.3.mavericks.bottle.tar.gz"
-    ruby_SHA="ab81211a2052ccaa6d050741c433b728d0641523d8742eef23a5b450811e5104"
+    ruby_URL="$HOMEBREW_BOTTLE_DOMAIN/bottles-portable-ruby/portable-ruby-2.6.3.yosemite.bottle.tar.gz"
```

Comment:
@mistydemeo This will need a revision bump and `Library/Homebrew/vendor/portable-ruby-version` bump if we want people to be force-upgraded 😭

Reply:
Done. Tested on 10.14 and the force-upgrade worked without a hitch. I've also bumped the Linux x86_64 bottle URL and dropped ARM bottles.